### PR TITLE
Fix master

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by jjanvier** [comment]
gtm

**Comment by grena** [comment]
GTM

File: features/Context/Page/User/Creation.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ public function __construct($session, $pageFactory, $parameters = array())
         $this->elements = array_merge(
             $this->elements,
             array(
-                'Roles' => array('css' => '#pim_user_user_form_roles_collection'),
+                'Roles' => array('css' => '#pim_user_user_form_rolesCollection'),
```
Comment (phaseinducer) [inline features/Context/Page/User/Creation.php]: [] please

**Comment by phaseinducer** [comment]
GTM

**Comment by solivier** [comment]
GTM

**Comment by ClementGautier** [comment]
GTM

**Pull request merged**